go/oasis-test-runner/oasis/fixture.go

```diff
@@ -239,6 +250,18 @@ func (f *ComputeWorkerFixture) Create(net *Network) (*Compute, error) {
 	})
 }
 
+// SentryFixture is a sentry node fixture.
+type SentryFixture struct {
+	Validators []int `json:"validators"`
+}
+
+// Create instantiates the client node described by the fixture.
+func (f *SentryFixture) Create(net *Network) (*Sentry, error) {
+	return net.NewSentry(&SentryCfg{
+		ValidatorIndices: f.Validators,
```

Comment:
Any reason why we don't resolve the indices here as we do with other things?

Reply:
This one won't work because sentries are created before validators and hence, it is not possible to resolve them at this time.

We could move the creation of validators before the creation of sentry nodes, but we would then be unable to resolve sentry nodes when creating new validators.

To sum up, one of two can resolve indices upon creation, the other needs to have indices.